Gemfile

```diff
@@ -40,6 +40,10 @@ gem "json", ">= 2.0.0"
 # Workaround until Ruby ships with cgi version 0.3.6 or higher.
 gem "cgi", ">= 0.3.6", require: false
 
+group :lint do
```

Comment:
If we add this group to the gemfile, does it get required automatically? Or do we need to add `required: false` here?

Reply:
Updated with `require: false` :+1: